config/crd/bases/anywhere.eks.amazonaws.com_cloudstackdatacenterconfigs.yaml

```diff
@@ -52,6 +52,10 @@ spec:
                   each other and a set of delegated administrators with some authority
                   over the domain and its subdomains
                 type: string
+              managementApiEndpoint:
```

Comment:
These changes are being introduced as part of e2e tests?

Reply:
These were missed in the previous PR, seems like make release-manifests was not run